cdatransform/transform/transform_lib/transform_with_YAML_v1.py

```diff
@@ -3,102 +3,112 @@
 from cdatransform.transform.validate import LogValidation
 
 
-def add_Specimen_rec(orig,MandT,DC,**kwargs):
-    cur_path = kwargs.get('cur_path',['cases','samples'])
-    spec_type = kwargs.get('spec_type','samples')
-    rel_path = kwargs.get('rel_path','cases.samples')
+def add_Specimen_rec(orig, MandT, DC, **kwargs):
```

Comment:
Can we use the variable DataCommons instead of DC across the transform project?  That and ManT just kind of seems not very descriptive to me.

Reply:
We can do that. It was just easier to type it all out since they appear very frequently. MandT is short for Mapping and Transform (dictionary)